src/styles/builder-ui.css

```
@@ -486,10 +492,13 @@ body.builder-body {
 
 /* Right edge positioning (default - Undo, Redo, Load, Save) */
 .workspace-edge-actions--right {
+  --workspace-edge-actions-offset-x: calc(100% - var(--builder-edge-actions-peek));
   right: calc(
-    clamp(8px, 2.25vw, 18px) + env(safe-area-inset-right, 0px)
+    env(safe-area-inset-right, 0px)
   );
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)